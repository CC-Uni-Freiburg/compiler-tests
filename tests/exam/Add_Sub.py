x = 0
while x + 80 - 120 + 42 - 69 + 50 < 0:
    print(x)
    x = x + 1
