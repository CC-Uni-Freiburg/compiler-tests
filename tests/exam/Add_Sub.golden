012345678910111213141516
